lib/xboxrt/libc_extensions/crtdbg.h

```diff
@@ -0,0 +1,11 @@
+#ifndef XBOXRT_CRTDBG
+#define XBOXRT_CRTDBG
+
+#include <assert.h>
+
+// Part of Microsoft CRT
+#define _ASSERT_EXPR(booleanExpression, message) assert(booleanExpression)
```

Comment:
What do you think about defining it as  `assert((booleanExpression) && (message))` instead to incorporate the message text when the assert fails?

Reply:
I'd be a bit paranoid over `NULL` - so I'd rather avoid such hacks which might suddenly do something unexpected.

The benefit of including `message` would be that logic would still run (if message was `message_list[i++]` for example).

However, I don't think anyone will care about these message for now anyway. They are merely to not fail compilation and catch the biggest issues - code will have to be looked at if they trigger anyway.

If we see them being used we can implement them properly (and avoid all the hacks).